setup.py

```diff
@@ -22,20 +22,20 @@
 from vsc.install.shared_setup import ag, kh, sdw, kw, wdp
 
 install_requires = [
-    'vsc-base >= 3.0.3',
-    'vsc-config >= 3.13.1',
-    'vsc-utils >= 2.0.0',
-    'pyyaml',
+    "vsc-base >= 3.0.3",
+    "vsc-config >= 3.13.1",
+    "vsc-utils >= 2.0.0",
+    "pyyaml <= 6.0.1",
```

Comment:
vsc-filesystems needs pyyaml?

Reply:
apparently it does. but you can't lock it like this. when making rpms for eg el9, this will need new rpms etc. for rpms, even for el6 it's not relevant etc etc.
there is a way to lock this via some vsc-install magic

Reply:
Here, right?
~~~
if sys.version_info < (3, 7):
            tests_requires.extend([
                'pyflakes~=2.3.0',
                'pycodestyle~=2.7.0',
                'pylint~=2.12.2',
                'prospector~=1.5.3.1',
                'flake8~=3.9.2',
                'pylint-plugin-utils < 0.7',
                'pylint-django~=2.4.4',
                # platformdirs >= 2.4.0 requires Python 3.7, use older versions for running tests with Python 3.6
                'platformdirs < 2.4.0',
                'typing-extensions < 4.2.0', # higher requires python 3.7
                'lazy-object-proxy < 1.8.0', # higher requires python 3.7
                'jsonpickle < 3.0.0', # higher requires python 3.7
                'importlib-metadata < 5.0.0', # no longer compatible with python 3.7
                'isort < 5.11.0',
                'zipp < 3.16', # no longer compatible with python 3.6
            ])
~~~~

Reply:
I fail to understand the rpm comment.